### PR TITLE
docs: fix Build badge in README

### PR DESCRIPTION
**Pull request by amis92**
I've just noticed the badge doesn't render correctly, so I went and copied the current markdown from GH Actions tab.

### CODE REVIEW

**Comment by gregsdennis** [comment]
Thanks.  I have a separate badge for test results.  Can you look into removing that from this one so that it just gives build status?

**Comment by amis92** [comment]
@gregsdennis it's just a fix because the old URL was broken. GitHub workflow badges don't support filtering on jobs inside, it'd need to be a custom badge as it is with tests.

**Comment by gregsdennis** [comment]
Ah, okay...  It looks like the title comes from the `name` specified at the top of the GH action .yml file.  I don't need the event spec, though.  That can be removed.

**Comment by amis92** [comment]
Depends what you want to see on the badge. Without the filter, it'll show latest run, for example for a pull request.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <br>
 
-[![Build status](https://github.com/gregsdennis/json-everything/workflows/.NET%20Core/badge.svg)](https://github.com/gregsdennis/json-everything/actions?query=workflow%3A%22Build+%26+Test%22)
+[![Build & Test](https://github.com/gregsdennis/json-everything/actions/workflows/dotnet-core.yml/badge.svg?event=push)](https://github.com/gregsdennis/json-everything/actions/workflows/dotnet-core.yml)
```
Comment (amis92) [inline README.md:7]:
```suggestion
[![Build & Test](https://github.com/gregsdennis/json-everything/actions/workflows/dotnet-core.yml/badge.svg?branch=master&event=push)](https://github.com/gregsdennis/json-everything/actions/workflows/dotnet-core.yml)
```

**Comment by gregsdennis** [comment]
Thanks for fixing this.

**Pull request merged**